torchtune/utils/attention_bias.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Couple comments on locations of things: (1) why is this stuff in `attention_bias.py`? This is just mask construction, right? We are not actually adding biases to the scores? And while I'm at it, (2) I wonder about the definition of `_attention_call` inside modules/attention.py. Call me crazy, but why not put all this in e.g. `torchtune/modules/attention_utils.py` or something